core/exception/system_call_error_spec.rb

```diff
@@ -96,11 +96,23 @@ def initialize
   end
 
   it "sets an 'unknown error' message when an unknown error number" do
-    SystemCallError.new(-1).message.should =~ /Unknown error(:)? -1/
+    platform_is_not :windows do
+      SystemCallError.new(-1).message.should =~ /Unknown error(:)? -1/
+    end
+
+    platform_is :windows do
+      SystemCallError.new(-1).message.should == "The operation completed successfully."
+    end
```

Comment:
Observed strange behaviour on Windows. CRuby calls `strerror` on Windows to get a string description for `errno` and it returns unexpected strings for unsupported errno codes (e.g. negative or not listed in https://learn.microsoft.com/en-us/cpp/c-runtime-library/errno-constants)